docs/spl-docs/common-usecases.md

```diff
@@ -248,4 +248,16 @@ This command determines the user's identity by checking the fields `username`, `
 
 **Problem:** In an inventory management system, there are instances where certain products need to be marked as discontinued or out of stock. For these products, resetting the `stock_level` field to NULL helps indicate that the field no longer holds any meaningful value and should be excluded from stock calculations.
 
-**Solution:** The [`null()`](evaluation-functions/comparison-conditional-functions.md#null) function can be used within an [`eval`](eval-command.md) expression to set the `stock_level` field to NULL for discontinued or out-of-stock products. [View full Solution](evaluation-functions/comparison-conditional-functions.md#use-case-example-8)
\ No newline at end of file
+**Solution:** The [`null()`](evaluation-functions/comparison-conditional-functions.md#null) function can be used within an [`eval`](eval-command.md) expression to set the `stock_level` field to NULL for discontinued or out-of-stock products. [View full Solution](evaluation-functions/comparison-conditional-functions.md#use-case-example-8)
+
+### 42. Analyzing Stock Market Data for Trends and Volatility
+
+**Problem:** A financial analyst needs to identify stocks with unusual trading patterns, focusing on high volatility, significant price movements relative to recent averages, and above-average trading volumes. This information can be used to spot potential market trends or anomalies that warrant further investigation.
+
+**Solution:** The [`streamstats`](streamstats-command.md) command can be used to calculate key metrics over a 5-day window. By applying various statistical functions, you can identify stocks with high volatility, significant price movements, and above-average trading volumes, thereby uncovering unusual trading patterns. [View full Solution](streamstats-command.md#analyzing-stock-market-data-for-trends-and-volatility)
```

Comment:
nit: this sounds like a fairly generic anomaly detection problem; I think rephrasing this as finding unusual trends/volatility with application usage (e.g., latency, number of active users, etc.) would probably be more applicable to most of our customers than finding unusual trends/volatility in the stock market

Reply:
addressed in the latest commit.